src/test/scala/org/broadinstitute/clio/webservice/AuthorizationWebServiceSpec.scala

```diff
@@ -0,0 +1,67 @@
+package org.broadinstitute.clio.webservice
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import akka.http.scaladsl.server.AuthorizationFailedRejection
+import org.scalatest.{FlatSpec, Matchers}
+import org.broadinstitute.clio.model._
+import org.broadinstitute.clio.service._
+
+class AuthWebServiceSpec extends FlatSpec
```

Comment:
Should the code also test the case where both `sub` and `user id` are provided, since it supports that case?